example/zfs-over-legacy.nix

```diff
@@ -48,6 +48,7 @@
           "root/zfs_fs" = {
             type = "zfs_fs";
             mountpoint = "/zfs_fs";
+            options.mountpoint = "legacy";
```

Comment:
No the idea is that ext4 is mounted with systemd and zfs will mount its own filesystem on top of that.